### DISCUSSION TITLE
Overleaf在使用在线方法解决生僻字后，“关键词”无法正常加粗

### DISCUSSION THREAD

**Discussion by Liangzheng-ZL** [sjtug/SJTUThesis#1004] [提问] [answered]
在使用https://github.com/sjtug/SJTUThesis/wiki/%E5%9C%A8%E7%BA%BF%E4%BD%BF%E7%94%A8%E8%AF%B4%E6%98%8E 中提供的方法解决了生僻字后，“关键词”无法加粗

**Comment by LogCreative** [comment, marked as answer]
已经更新 Wiki，请使用新的导言区设定。
